compiler/rustc_const_eval/src/transform/check_consts/ops.rs

```diff
@@ -297,10 +297,12 @@ impl<'tcx> NonConstOp<'tcx> for FnCallNonConst<'tcx> {
             ccx.const_kind(),
         ));
 
-        if let Some(feature) = feature
-            && ccx.tcx.sess.is_nightly_build()
-        {
-            err.help(format!("add `#![feature({feature})]` to the crate attributes to enable",));
+        if let Some(feature) = feature {
+            ccx.tcx.disabled_nightly_feature(
+                &mut err,
+                body.source.def_id().as_local().map(|local| ccx.tcx.local_def_id_to_hir_id(local)),
+                vec![(String::new(), feature)].into_iter(),
```

Comment:
```suggestion
                [(String::new(), feature)],
```
tip: you don't need vectors to create iterators, arrays implement `IntoIterator` :)